### DISCUSSION TITLE
Creating Jobs

### DISCUSSION THREAD

**Discussion by paultvis** [atrocore/atrocore#570] [General]
Hi,

Is there any documentation available on creating jobs? We have a number of small queries and cleanup activities we would like to run as scheduled jobs but I cannot see anywhere how these are created.

Thanks

Paul

**Comment by rratsun** [comment]
Hello,

We have forgotten to create documentation for this :) We will write the documentation very soon.
But for now you should do:
* Create Job class in Jobs dir. Looks like [this](https://github.com/atrocore/atropim/blob/master/app/Jobs/PimCleanup.php). After creation you will see new job here 
![image](https://user-images.githubusercontent.com/33658723/208376590-cd81b3fe-3bf1-4821-9e3f-9323029d8185.png)

* If you want to add translation label you should do like [here](https://github.com/atrocore/atrocore/blob/master/app/Espo/Resources/i18n/en_US/ScheduledJob.json). This part ``` "TreoCleanup": "Old deleted data cleanup",```
